persistence-modules/pom.xml

```diff
@@ -28,6 +28,7 @@
         <module>java-jdbi</module>
         <module>java-jpa</module>
         <module>java-mongodb</module>
+        <module>elasticsearch</module>
```

Comment:
Please move this line so that the modules are in alphabetical order by module name.